kong/conf_loader/utils.lua

```diff
@@ -0,0 +1,925 @@
+local require = require
```

Comment:
Are we next going to split this utils?

Or should we do it already, aka move things in `kong.tools.xxxx`?

Reply:
I think that these functions are related to conf_loader only, they will not be used by other modules,
and there are already some `router/utils.lua` `clustering/utils.lua`.